HEN_HOUSE/egs++/egs_advanced_application.cpp

```diff
@@ -406,6 +406,20 @@ EGS_AdvancedApplication::~EGS_AdvancedApplication() {
     if (n_rng_buffer > 0) {
         delete [] rng_buffer;
     }
+    if (nmed > 0) {
```

Comment:
Interpolators are allocated if `nmed > 0` in `EGS_AdvanchedApplication::helpInit` but not deallocated.https://github.com/nrc-cnrc/EGSnrc/blob/3e0db683e48a328b8fede9e7756e0deb693fbf9e/HEN_HOUSE/egs%2B%2B/egs_advanced_application.cpp#L661-L672